tests/v2/validation/snapshot/snapshot.go

```diff
@@ -362,14 +362,6 @@ func restoreV2Prov(t *testing.T, client *rancher.Client, v2prov initialSnapshotC
 		assert.Empty(t, podErrors)
 		require.Equal(t, v2prov.kubernetesVersion, clusterObject.Spec.KubernetesVersion)
 
-		steveclient, err := client.Steve.ProxyDownstream(clusterID)
-		require.NoError(t, err)
-
-		deploymentList, err := steveclient.SteveType(workloads.DeploymentSteveType).NamespacedSteveClient(defaultNamespace).List(nil)
-		require.NoError(t, err)
-		require.Equal(t, 1, len(deploymentList.Data))
-		require.Equal(t, initialWorkloadName, deploymentList.Data[0].ObjectMeta.Name)
```

Comment:
why is this removed? k3s is seeing a failure, not sure if this is related or not

Reply:
that failure isn't related. This is being removed because tbh it shouldn't have been here to begin with. The check doesn't make any sense, we care to remove the workloads after restore happened. RKE1 doesn't check this, v2 prov shouldn't either.